com.unity.render-pipelines.high-definition/Runtime/Material/AxF/AxF.shader

```diff
@@ -135,15 +170,17 @@ Shader "HDRP/AxF"
 
         [ToggleUI] _SupportDecals("Support Decals", Float) = 1.0
         [ToggleUI] _ReceivesSSR("Receives SSR", Float) = 1.0
-
+        [ToggleUI] _ReceivesSSRTransparent("Receives SSR Transparent", Float) = 0.0
         [ToggleUI] _AddPrecomputedVelocity("AddPrecomputedVelocity", Float) = 0.0
 
     }
 
     HLSLINCLUDE
 
+    #pragma prefer_hlslcc gles
+    #pragma exclude_renderers d3d11_9x gles
     #pragma target 4.5
-    #pragma only_renderers d3d11 playstation xboxone vulkan metal switch
+    //#pragma only_renderers d3d11 playstation xboxone vulkan metal switch
```

Comment:
?

Reply:
What is this?
  #pragma prefer_hlslcc gles
    #pragma exclude_renderers d3d11_9x gles